dist/ExtUtils-ParseXS/lib/perlxstut.pod

```diff
@@ -1382,20 +1401,22 @@ Some systems may have installed Perl version 5 as "perl5".
 =head1 See also
 
 For more information, consult L<perlguts>, L<perlapi>, L<perlxs>, L<perlmod>,
-and L<perlpod>.
+L<perlapio>, and L<perlpod>.
 
 =head1 Author
 
-Jeff Okamoto <F<okamoto@corp.hp.com>>
+Jeff Okamoto
```

Comment:
Why remove this email?

Reply:
I notified the individual of its removal and was returned a thank you.